bitnami/grafana-operator/values.yaml

```
@@ -418,6 +418,13 @@ grafana:
       #   # allowed_domains: example.com
       #   allow_sign_up: True
 
+  ## Mounting extra files into the Grafana Pod
+  ## Ref: https://github.com/integr8ly/grafana-operator/blob/master/documentation/extra_files.md
+  ##
+  confiMaps: {}
```

```suggestion
  configMaps: []
```

As it is an array. Typo as well.

---

@alemorcuq  but if I set it as array will this code work ??

```
{{ toYaml .Values.grafana.configMaps | nindent 4 }}
```

---

Did work! I thought It was necessary to use range when using arrays.